### PR TITLE
Add more Rust crates

### PR DESCRIPTION
**Pull request by rushmorem**

### CODE REVIEW

**Review by dnsguru — APPROVED** [review]
LGTM, Submitted by primary author of initial Rust Lib = ++trusty

**Review by cklim5198 — COMMENTED** [review]
Updated

**Comment by cklim5198** [comment]
> Updated

File: learn/index.html
Comment (cklim5198) [inline learn/index.html:1]: Updated

**Review by cklim5198 — COMMENTED** [review]
Updated

**Pull request merged**